<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTask</class>
 <widget class="QDialog" name="DialogTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Nome task</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelPriority">
     <property name="text">
      <string>Priorit√†</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>Descrizione</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="layoutDueDate">
     <item>
      <widget class="QCheckBox" name="enableDueDate">
       <property name="toolTip">
        <string>Abilita campo scadenza</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="fieldDueDate">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="buttonLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="okCancelButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="fieldLocation"/>
   </item>
   <item row="3" column="1">
    <widget class="QSlider" name="prioritySlider">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>3</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBelow</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="labelLocation">
     <property name="text">
      <string>Luogo</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelDueDate">
     <property name="text">
      <string>Scadenza</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="fieldTaskName"/>
   </item>
   <item row="5" column="1">
    <widget class="QTextEdit" name="fieldDescription">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="labelCompleted">
     <property name="text">
      <string>Completato</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="taskCompleted">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>accepted()</signal>
   <receiver>DialogTask</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okCancelButtons</sender>
   <signal>rejected()</signal>
   <receiver>DialogTask</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogTask</receiver>
   <slot>deleteEvent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enableDueDate</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogTask</receiver>
   <slot>enableDueDate(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>124</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteEvent()</slot>
  <slot>enableDueDate(int)</slot>
 </slots>
</ui>
